sdcm/tester.py

```diff
@@ -2106,6 +2106,7 @@ def verify_stress_thread(self, cs_thread_pool):
         errors = errors[-5:]
         if errors:
             self.log.warning("cassandra-stress errors on nodes:\n%s", "\n".join(errors))
+        return results and not errors
```

Comment:
I'm not sure all the errors you've see during restore nemesis, are part of what `verify_results` would identify. (it's not exactly as what we use for generating events (maybe it should)

need to do a run or two to validate

Reply:
In the results I see, that the stress return with return value of 1, is enough to make this work.

so it's a start, but not covering all possible case of failure during the stress.